rallyrolebot/cogs/defaults_cog.py

```diff
@@ -0,0 +1,86 @@
+import json
+import sys
+import traceback
+
+import discord
+from discord.ext import commands, tasks
+from discord.utils import get
+
+import data
+import rally_api
+import validation
+import errors
+
+from cogs import update_cog
+
+from constants import *
+from utils import pretty_print
+
+
+class DefaultsCommands(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    async def cog_after_invoke(self, ctx):
+        await pretty_print(
+            ctx, "Command completed successfully!", title="Success", color=SUCCESS_COLOR
+        )
+
+    @errors.standard_error_handler
+    async def cog_command_error(self, ctx, error):
+        # All other Errors not returned come here. And we can just print the default TraceBack.
+        print("Ignoring exception in command {}:".format(ctx.command), file=sys.stderr)
+        traceback.print_exception(
+            type(error), error, error.__traceback__, file=sys.stderr
+        )
+
+    @commands.command(
+        name="set_default_coin",
+        help=" <coin name> Set a default coin to be used across the server",
+    )
+    @validation.owner_or_permissions(administrator=True)
+    async def set_default_coin(self, ctx, coin_name):
```

Comment:
#1 mentions that this should warn the user and prompt for confirmation.

Reply:
Done